frontend/src/components/Main/DatasetModelOptionList/DatasetModelOptionList.js

```diff
@@ -38,7 +38,13 @@ class DatasetModelOptionList extends Component {
                             <th>Adverse Direction</th>
                         </tr>
                     ) : null}
-                    {dtype == Dtype.DICHOTOMOUS || dtype == Dtype.MULTI_TUMOR ? (
+                    {dtype == Dtype.MULTI_TUMOR ? (
```

Comment:
for multitumor, do we need to set a degree for each model? this may be ok for now, but i wonder if we'll get that feedback.

Reply:
also, remove the dtytpe check with the `dtype == Dtype.DICHOTOMOUS` block below